src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/ExperimentResource.java

```diff
@@ -368,7 +387,8 @@ private Experiment updateExperimentGapStop(int id, Experiment experiment, Experi
         newPopulations.forEach(population -> {
             try {
                 platformManager.publishTask(population.getPlatformId(), old).join();
-                storePopulation(id, population);
+                //TODO marcel: update?
```

Comment:
ok, but platformManager.publishTask already modifies the database, also the database has an published_failed status which gets set if the publishing failed.